alicloud/resource_alicloud_ots_instance_test.go

```diff
@@ -381,124 +277,118 @@ func TestAccAlicloudOtsInstanceHighPerformance_updateAccessBy(t *testing.T) {
+			testAccPreCheck(t)
+			testAccPreCheckWithRegions(t, false, connectivity.OtsCapacityNoSupportedRegions)
 		},
-
 		// module name
 		IDRefreshName: resourceId,
 		Providers:     testAccProviders,
-		CheckDestroy:  testAccCheckOtsInstanceDestroy,
+		CheckDestroy:  rac.checkResourceDestroy(),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccOtsInstance(string(OtsHighPerformance), rand),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckOtsInstanceExist("alicloud_ots_instance.foo", &instance),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "name", fmt.Sprintf("tf-testAcc%d", rand)),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "accessed_by", "Any"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "instance_type", string(OtsHighPerformance)),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "description", fmt.Sprintf("tf-testAcc%d", rand)),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.%", "2"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.Created", "TF"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.For", "acceptance test"),
-				),
-			},
-			{
-				ResourceName:      resourceId,
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
-			{
-				Config: testAccOtsInstanceUpdateAll(string(OtsHighPerformance), rand),
+				Config: testAccConfig(map[string]interface{}{
+					"name":          name + "${count.index}",
+					"description":   name + "${count.index}",
+					"instance_type": "Capacity",
+					"count":         "5",
+				}),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckOtsInstanceExist("alicloud_ots_instance.foo", &instance),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "name", fmt.Sprintf("tf-testAcc%d", rand)),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "accessed_by", "ConsoleOrVpc"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "instance_type", string(OtsHighPerformance)),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "description", fmt.Sprintf("tf-testAcc%d", rand)),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.%", "3"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.Updated", "TF"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.For", "acceptance test"),
-					resource.TestCheckResourceAttr("alicloud_ots_instance.foo", "tags.From", "TF"),
+					testAccCheck(nil),
 				),
 			},
 		},
 	})
+}
 
+func resourceOtsInstanceConfigDependence(name string) string {
+	return ""
+}
+
+var otsInstanceBasicMap = map[string]string{
+	"name":          CHECKSET,
+	"accessed_by":   "Any",
+	"instance_type": CHECKSET,
+	"description":   CHECKSET,
 }
 
 func testAccCheckOtsInstanceExist(n string, instance *ots.InstanceInfo) resource.TestCheckFunc {
```

Comment:
please remove this

Reply:
used in other testcases, it will be removed after all ots pr merged.